docs/operations/memory_dump.md

```diff
@@ -13,9 +13,16 @@ The memory dump process mounts a PVC to the virt-launcher in order to get the ou
 [feature gates](./activating_feature_gates.md#how-to-activate-a-feature-gate)
 field in the KubeVirt CR must be expanded by adding the `HotplugVolumes` to it.
 
-### A PVC to hold the memory dump output
+## Virtctl support
+
+### Get memory dump
+
+Now lets assume we have a running VM and the name of the VM is 'my-vm'.
+We can either dump to an existing pvc, or request one to be created.
 
-Currently in order to get the memory dump output a pre-existing PVC is required. The size of the PVC must be big enough to hold the memory dump (The VM memory size + 100Mi overhead).
+#### Existing PVC
+The size of the PVC must be big enough to hold the memory dump (The VM memory size + 100Mi overhead multiplied with the filesystem overhead),
```

Comment:
this `The VM memory size + 100Mi overhead multiplied with the filesystem overhead,` might be a little bit ambiguous. I think I know what it means, but I can also read it as `memSize + (100Mi * fsOverhead)` or (memSize +  100Mi) * fsOverhead. 

And which one is correct? And why 100Mi?

Reply:
the correct one is 

> (memSize + 100Mi) * fsOverhead

100Mi is a const overhead of the memorydump there is no actual explanation.. added of some headers and stuff..
How would you think is best writing so it will be clear.. its already in braces () so should I write what you did? maybe without the fs and mem? i.e:
( (VMMemorySize+100Mi) * FilesystemOverhead )
I wanted to put it more in words then in a formula but if its not clear then ok.

Reply:
Maybe something like this, if not too verbose. Just change *stuff* with some other word that better describes it, are these logs or utils? 

The size of the PVC must be big enough to hold the memory dump. The calculation is `( (VMMemorySize+ MemoryDumpOverhead) * FilesystemOverhead )`, where `VMMemorySize` is the size of VM memory, `MemoryDumpMetadata` is the space reserved for memory dump headers and some *stuff*, `FilesystemOverhead` is the value used to adjust requested PVC size withe filesystem overhead.

Reply:
@brybacki added a new commit to rephrase this explanation